src/helix-ui/styles/base/links.less

```diff
@@ -0,0 +1,16 @@
+a[href] {
```

Comment:
The selector was changed from `a` to `a[href]` because the styles should only be applied to actual links (i.e. they navigate away from the page).